### DISCUSSION TITLE
How to rebuild

### DISCUSSION THREAD

**Discussion by madferro** [awcodes/filament-tiptap-editor#530] [Q&A]
Hello, i forked this repo, i added some parameters in the Link extension and in the `resources/js/plugin.js` file, but these changes are not present on the `public/js/awcodes/tipatap-editor/components/tiptap.js` minified file.
I guess I have to rebuild it, i tried with 
`npm run build`
 but it doesn't work.
 
Can you please help me?
Thank you in advance!

**Comment by awcodes** [comment]
You would have to rebuild the plugin js but also republish the assets with `artisan filament:assets`

**Reply by madferro** [reply]
Sorry but i can't understand how to rebuild the plugin. Can you please help me?
Thank you in advance

**Reply by drifteaur** [reply]
just run `artisan filament:assets` after running `npm run build` in the package directory
